### ISSUE TITLE
Modified colorschemes fail to load

### ISSUE THREAD

**Issue by pulyavin** [vprimachenko/Sublime-Colorcoder#43]
Hello, after unistal plugin, sublime alert this message: "Error loading colour scheme Packages/Colorcoder/Monokai (Colorcoded).tmTheme: Unable to open Packages/Colorcoder/Monokai (Colorcoded).tmTheme"

**Comment by vprimachenko** [comment]
OS? Version? how installed?

**Comment by lelandsmith** [comment]
I got the same thing. Any ideas on how to fix this?

Sublime: 3083
OS: Mac 10.11 El Capitan

**Comment by husamuddin** [comment]
The same here :cry:

Sublime: 3083
OS: Ubuntu 14.04

**Comment by mcwqy9** [comment]
This should be reopened, as it is a bug in this package not cleaning up after itself. To fix this issue, I manually edited my user settings.

On OSX, command-, delete the line about "color_scheme":  "Packages/Color Scheme - Default/Monokai (Colorcoded).tmTheme"

**Comment by vprimachenko** [comment]
@mcwqy9 are you sure the line was "Packages/Color Scheme - Default/Monokai (Colorcoded).tmTheme"? Modified themes should land in colorcoder own folder. Could you also upload the theme file in question, i would need to inspect it wether Colorcoder failed to modify it properly or something else happened

**Comment by Vladislav-Shevtsov** [comment]
Yes guys I have the same problem after unistaling the same message appear....

**Comment by vkn999** [comment]
Vladislav-Shevtsov, try to change the theme

**Comment by bitxpert** [comment]
This is real pain, i wonder why this issue is still there after that much time.

**Comment by matyushkin** [comment]
The same problem
